### ISSUE TITLE
I don't think we are releasing subscription, etc. objects in all cases.

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#632]
In testing for update of the scriptlets in pr 623 I came to the conclusion that we may not be correctly releaseing instances of filters, etc.  I am going to have to run a detailed test and review the run_cim_operations.py code today.  I ran run_cim_operations.py a few minutes ago and there were a number of subscriptions/filters/destinations still in the server.  If it is a problem in the test, OK. However, it may be a problem in the server so I want to be sure of this before we release.

**Comment by KSchopmeyer** [comment]
I went back and executed all of the run_cim_operations.py tests for subscriptions and thee entire test repeatedly and we ARE releasing the subscriptions, etc. in all cases.  What I saw the other day I don't understand except that I cannot duplicate it.

Not sure what is happening here except that I have seen two cases in other testing where we left lots of indications in the server.  

At this point I think I have to marke this as invalid and close it.
